packages/table/src/Table.js

```diff
@@ -529,39 +529,44 @@ const headers = tableConfig => tableConfig.fields
   .filter(field => !field.disappear)
   .map(field => getHeaderCell(field))
 
-const isRowLoaded = (tableData, loadLookAhead) => ({ index }) => {
-  if (Array.isArray(tableData)) return !!tableData[index + loadLookAhead]
```

Comment:
Checking a different index to see if a row has been loaded could cause some rows to be loaded multiple times.